tox.ini

```
@@ -58,6 +58,7 @@ deps =
     quadprog >=0.1.11
     scipy >=1.2.0
     scs >=3.2.0
+    sip >=0.0.1
```

```suggestion
    sip-python >=0.0.1
```

Since the PyPI package is https://pypi.org/project/sip-python/